app/experimenter/experiments/models/nimbus.py

```diff
@@ -436,14 +439,29 @@ def latest_review_request(self):
         return (
             self.all()
             .filter(
-                old_status=NimbusExperiment.Status.DRAFT,
+                Q(old_status=NimbusExperiment.Status.DRAFT)
+                | Q(old_status=NimbusExperiment.Status.PREVIEW),
```

Comment:
I had to think this through for a sec since when the experiment is being reviewed it'll be in draft, but this seems right since IIUC a review request can _occur_ when it's in `PREVIEW`.

Reply:
Actually we might be able to remove this part of the filter altogether, now that we have publish status we don't have to look at transitions of status>status to look for review requests, this might be sufficient:

```py
                old_publish_status=NimbusExperiment.PublishStatus.IDLE,
                new_status=NimbusExperiment.Status.DRAFT,
                new_publish_status=NimbusExperiment.PublishStatus.REVIEW,
```